### DISCUSSION TITLE
Ways to support bun development?

### DISCUSSION THREAD

**Discussion by vestimir** [oven-sh/bun#702] [General]
@Jarred-Sumner what can individuals or companies do to help at this stage of the project? 

PS: Bun is great 🙌

**Comment by jub0t** [comment]
Probably use it, the more people use it the faster more bugs can be found and the faster Bun gets stable.

**Comment by vestimir** [comment]
We have a ton of Next.js projects with varying levels of complexity + many internal NPM packages. I'll try to run them with bun and report what I've found.
